generic_config_updater/services_validator.py

```diff
@@ -48,6 +48,18 @@ def _service_restart(svc_name):
     return rc == 0
 
 
+def rsyslog_validator(old_config, upd_config, keys):
+    old_syslog = old_config.get("SYSLOG_SERVER", {})
+    upd_syslog = upd_config.get("SYSLOG_SERVER", {})
+
+    if old_syslog != upd_syslog:
```

Comment:
>!= [](http://example.com/codeflow?start=18&length=2)

Use the native recursive comparison is working in this case, but is considered dangerous.

By json protocol, the order in a list matters, but by ConfigDB schema, the order in a list does not matter. For example /EVERFLOW/ports is a list.

Could you use a general ConfigDB aware json recurisve comparison? Or in this case, just use a ConfigDB/SYSLOG_SERVER specific comparison. check example in `vlan_validator()`.